### ISSUE TITLE
E3002 Invalid Property Lambda/Properties/EphemeralStorage

### ISSUE THREAD

**Issue by waldofouche** [aws-cloudformation/cfn-lint#2248]
*cfn-lint version: (`cfn-lint --version`)*

`cfn-lint 0.58.4`

*Description of issue.*
Looks like it does not yet recognize `EphemeralStorage` as a valid property for lambdas
```yml
Lambda:
    Type: AWS::Lambda::Function
    Properties:
      Role: !GetAtt Role.Arn
      Timeout: 600
      MemorySize: 2048
      EphemeralStorage:
        Size: 1024
```

Link to the [**docs**](https://aws.amazon.com/blogs/compute/using-larger-ephemeral-storage-for-aws-lambda/) where it shows the new feature

Cfn-lint uses the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) as the base to do validation. These files are included as part of the application version. Please update to the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

**Comment by kddejong** [comment]
Included in #2245.  Will be fixed on the release coming shortly.

Should be able to run `cfn-lint -u` to update the specs prior to the release.
